commons/java/openjdk8/Dockerfile

```diff
@@ -2,5 +2,3 @@ FROM ubuntu:18.04
 
 RUN apt-get update -y && apt-get install -y --no-install-recommends openjdk-8-jdk-headless \
 	&& rm -rf /var/lib/apt/lists/*
-
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
```

Comment:
I believe this change is not needed for this PR, right?
If yes, can you please revert the changes to this file?
Thanks!